knowrob_kautham/src/kauthamwrapper.cpp

```diff
@@ -0,0 +1,93 @@
+#include "knowrob_kautham/knowrob_kautham.h"
+
+void ham_product(double ax, double ay, double az, double aw, double bx, double by, double bz, double bw, double &rx, double &ry, double &rz, double &rw)
```

Comment:
what is "ham", what is this function doing?

Reply:
quaternion product

Reply:
is this common notation? never heard it :)